src/components/Paginate.tsx

```diff
@@ -21,7 +21,7 @@ export default function Paginate({
   totalResults,
 }: paginateProps) {
   const [currentPage, setCurrentPage] = useState(page);
-  const numberOfPages: number = findNumberOfPages(totalResults, limit);
+  const numberOfPages: number = getPageCount(totalResults, limit);
```

Comment:
We renamed the function that finds the number of pages of search results from `findNumberOfPages` to `getPageCount` because `findNumberOfPages` implies use of the .find() method [as amilajack mentioned](https://github.com/amilajack/tide/pull/290#discussion_r546048793).